.github/workflows/macOS.yml

```diff
@@ -59,4 +59,4 @@ jobs:
         brew reinstall boost cmake cpputest dbus jsoncpp ninja
     - name: Build
       run: |
-        OTBR_OPTIONS='-DOTBR_BORDER_AGENT=OFF -DOTBR_MDNS=OFF -DOT_FIREWALL=OFF' ./script/test build
+        OTBR_OPTIONS='-DOTBR_BORDER_AGENT=OFF -DOTBR_MDNS=OFF -DOT_FIREWALL=OFF -DOTBR_DBUS=OFF' ./script/test build
```

Comment:
Why turn it off?

Reply:
That's because now D-BUS server has a dependency on Publisher.